code/common/finspace.q

```diff
@@ -55,3 +55,12 @@ newrdbup:{[]
       .lg.o[`newrdbup;"received signal from next period rdb, setting rdbready to true"];
       @[`.finspace;`rdbready;:;1b];
  };
+
+ // clustername must be of type string
+deletecluster:{[clustername]
+  if[10h<>fType:type clustername; .lg.o[`deletecluster;"clustername must be of type string: 10h, got ",-3!fType]; :(::)];
```

Comment:
if we get a symbol, can we just `string` it here rather than throwing an error? (fine to throw an error for other datatypes, and I think this should probably be an error log rather than out log? i.e. `.lg.e`)

Reply:
resolved